ee/modules/610-service-with-healthchecks/crds/doc-ru-servicewithhealthchecks.yaml

```
@@ -46,152 +44,158 @@ spec:
+                          
                                   Имя должно соответствовать `IANA_SVC_NAME`.
                           tcp:
                             properties:
                               targetPort:
                                 description: |-
                                   Имя или номер порта для доступа к контейнеру.
-                                  Номер должен находиться в диапазоне от 1 до 65535.
+                                  
                                   Имя должно быть именем `IANA_SVC_NAME`.
                 allocateLoadBalancerNodePorts:
                   description: |-
-                    `allocateLoadBalancerNodePorts` определяет, будут ли автоматически выделяться NodePorts для сервисов с типом LoadBalancer.
+                     Определяет, следует ли автоматически выделять NodePort для сервисов типа LoadBalancer.
                      Значение по умолчанию — `true`. Значение можно установить в `false`, если балансировщик нагрузки кластера не зависит от NodePorts.
                      
-                     Если вызывающий запросит конкретные NodePorts (указав значение), эти запросы будут соблюдены независимо от этого поля.
-                     Это поле может быть установлено только для сервисов с типом LoadBalancer и будет очищено, если тип изменится на любой другой.
+                     Если запрашивающая сторона указывает конкретные значения NodePort, эти запросы будут выполнены независимо от значения этого параметра. 
+                     Данный параметр может быть установлен только для сервисов типа LoadBalancer и автоматически сбрасывается при изменении типа на другой.
                 clusterIP:
                   description: |-
-                    `clusterIP` — это IP-адрес сервиса, который обычно назначается случайным образом.
-                     Если адрес указан вручную, он должен быть в пределах допустимого диапазона (в соответствии с конфигурацией системы) и не быть занятым, чтобы быть выделенным для сервиса;
-                     в противном случае создание сервиса завершится неудачей. Это поле нельзя изменять через обновления, если поле типа также не изменяется на ExternalName (что требует, чтобы это поле было пустым)
-                     или если поле типа изменяется с ExternalName (в этом случае это поле может быть указано дополнительно, как описано выше). 
-                     
-                     Допустимые значения: `None`, пустая строка (""), или действительный IP-адрес. Установка этого значения в `None` создает сервис без виртуального IP, что полезно, когда предпочтительнее прямое подключение к конечным точкам,
-                     и проксирование не требуется. Это применяется только к типам ClusterIP, NodePort и LoadBalancer. Если это поле указано при создании сервиса типа ExternalName, создание завершится неудачей. 
+                     Список IP-адресов, назначенных сервису.
```

```suggestion
                     IP-адрес сервиса.
```
Это не список, как в clusterIPs.